app/src/main/java/com/example/inventory/ui/home/HomeScreen.kt

```diff
@@ -103,31 +102,35 @@ fun HomeScreen(
         HomeBody(
             itemList = homeUiState.itemList,
             onItemClick = navigateToItemUpdate,
-            modifier = modifier
-                .padding(innerPadding)
-                .fillMaxSize()
+            contentPadding = innerPadding,
```

Comment:
nit: As you are adding contentPadding to the function signature, put contentPadding after the modifier to be consistent with the new function signature.

Reply:
The reason for this is Modifier should be the first optional parameter, but for function calls we usually send Modifier as the last parameter. I checked some samples and they are inconsistent. 

https://github.com/android/compose-samples/blob/2410841f0cd2384789be0ad48528cabfbf097db2/Reply/app/src/main/java/com/example/reply/ui/components/ReplyEmailThreadItem.kt#L95